traitcuration/traits/templates/traits/trait_detail.html

```diff
@@ -236,11 +236,63 @@ <h4>New Term Information</h4>
       </div>
     </section>
     {% endif %}
+
+    <section class="section">
+      <h4>TRAIT HISTORY</h4>
+
+      {% if not user.is_anonymous %}
+      <div class="post-comment__container">
+        <div class="post-comment__header">
+          <span uk-icon="user"></span>
+          <span>Comment as {{user.get_full_name}}</span>
+          <button class="uk-button button button--primary post-comment__submit" onclick="commentButtonClicked()">Submit</button>
+        </div>
+
+        <div class="post-comment__body">
+          <textarea 
+          id="commentBody" 
+          class="uk-textarea post-comment__textarea" 
+          placeholder="Leave a commment..."></textarea>
+        </div>
+      </div>
+      {% endif%}
+
+      {% for event in history_events|dictsortreversed:"date" %}
+      {% if event.type == 'comment' %}
+      <div class="event event--comment">
+        <span class="event__title">{{event.content.author.get_full_name}} commented</span>
+        <span date="{{event.date.isoformat}}" class="date event__date"></span>
+        <p>{{event.content.body}}</p>
+      </div>
+
+      {% elif event.type == 'review' %}
+      <div class="event">
+        <p class="event__title"> 
+          {{event.content.reviewer.get_full_name}} reviewed mapping of trait "{{trait.name}}"
+          to term "{{event.content.mapping_id.term_id.curie}} - {{event.content.mapping_id.term_id.label}}"
+        </p>
+        <span date="{{event.date.isoformat}}" class="date event__date"></span>
+      </div>
+
+      {% elif event.type == 'mapping' %}
+      <div class="event"">
+        <p class="event__title"> 
+          {{event.content.curator.get_full_name}} mapped trait 
+          "{{trait.name}}" to term "{{event.content.term_id.curie}} - {{event.content.term_id.label}}"
+        </p>
+        <span date="{{event.date.isoformat}}" class="date event__date"></span>
+      </div>
+
+      {% endif %}
+
+      {% endfor %}
+    </section>
```

Comment:
Finally, there appears to be this strange bug where the timestamp of the comment does not actually reflect the time when I leave it, and instead reflects the time the app was first launched (?)

For example, I started reviewing this PR at 23:08 and opened the deployment. Since then, all comments which I leave have the timestamp of 23:08, for example:

![image](https://user-images.githubusercontent.com/10669118/89111523-8f018300-d44e-11ea-9d58-951121d65964.png)